lib/screens/tree_dict/dict_card.dart

```diff
@@ -0,0 +1,60 @@
+import 'package:flutter/material.dart';
+
+class DictCard extends StatelessWidget {
+  final String imageURL;
+  final String commonName ;
+  final String scientificName ;
+
+  DictCard(this.scientificName,this.commonName,this.imageURL);
+
+  @override
+  Widget build(BuildContext context) {
+    return Container(
+      padding: EdgeInsets.fromLTRB(10, 10, 10, 0),
+      height: 200,
+      width: double.maxFinite,
+      child: Card(
+        shape: RoundedRectangleBorder(
+            borderRadius: BorderRadius.all(Radius.circular(15))),
+        elevation: 10,
+        child: Row(
+          children: [
+            Container(
+              width: MediaQuery.of(context).size.width/3,
+              child: Image(
+                image: NetworkImage(imageURL),
+                fit: BoxFit.fill,
+              ),
+            ),
+            Padding(
+              padding: const EdgeInsets.all(8.0),
+              child: Column(
+                mainAxisAlignment: MainAxisAlignment.start,
+                crossAxisAlignment: CrossAxisAlignment.start,
+                children: [
+                  Text(
+                    commonName,
+                    style: TextStyle(
+                      fontSize: 30,
+                      color: Colors.black,
+                      fontFamily: "Poppins",
```

Comment:
And you didn't added in pubspec too

Reply:
I created it for making the builder for ListView. But went with doing it in the plictionary.dart file

Reply:
I didn't add that font. It was already being used.

Reply:
[From your previous commit](https://github.com/ashwinkey04/lorax/blob/29082792a9524a9954a93133ed94dbefadcfc74d/lib/screens/plictionary.dart#L86)

Reply:
okay, we might have missed them 
next time we will add them no worries 

it will not show error it will take the default one